Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node('docker') {
     stage('Build') {
         dockerCleanup()
         docker.build(image.id, '--pull --no-cache .')
-        sh "docker tag ${image.id} docker-registry.usersys.redhat.com/${image.id}"
+        sh "docker tag ${image.id} registry.devshift.net/${image.id}"
```

Comment:
Does this apply? 

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.migrations#L1

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/hack/generate-db-migrations.sh#L8

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/runtest.sh#L15